build/nsis.install.nsh

```diff
@@ -19,7 +19,7 @@ Section "Geth" GETH_IDX
 
   # Create start menu launcher
   createDirectory "$SMPROGRAMS\${APPNAME}"
-  createShortCut "$SMPROGRAMS\${APPNAME}\${APPNAME}.lnk" "$INSTDIR\geth.exe" "--fast" "--cache=512"
+  createShortCut "$SMPROGRAMS\${APPNAME}\${APPNAME}.lnk" "$INSTDIR\geth.exe" "--syncmode fast" "--cache=512"
```

Comment:
Both `syncmode=fast` and `cache=512` should not be needed. I suggest removing them. The former is default, and for the latter, wouldn'it it be better to use whatever the default is?